src/SmartComponents/DriftPage/DriftTable/ReferenceSelector/ReferenceSelector.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { OutlinedStarIcon, StarIcon } from '@patternfly/react-icons';
+import PropTypes from 'prop-types';
+
+class ReferenceSelector extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isReference: this.props.isReference
+        };
+    }
+
+    render() {
+        const { updateReferenceId, id } = this.props;
+        const { isReference } = this.state;
+
+        return (
+            isReference
+                ? <StarIcon
+                    className='reference-selector pointer'
+                    onClick={ () => updateReferenceId(undefined) }
+                />
```

Comment:
Should this be indented one more step to align with the `<StarIcon` ?

Reply:
No, the `<StarIcon` isn't indented, it's just a space between the `?` and `<StarIcon`